src/crimson/os/seastore/async_cleaner.h

```diff
@@ -75,6 +76,14 @@ struct segment_info_t {
 
   void set_closed();
 
+  void init_modify_time(sea_time_point _modify_time, std::size_t _num_extents) {
```

Comment:
Let's drop `_num_extents` which is not used, keeping `ceph_assert(num_extents == 0);` should be good.